python/src/JointDistribution_doc.i.in

```diff
@@ -6,22 +6,24 @@ Parameters
 distributions : list of :class:`~openturns.Distribution`
     List of :math:`n` marginals of the distribution. Each marginal must be of
     dimension 1.
-copula : :class:`~openturns.Distribution`, optional
-    A copula. If not mentioned, the copula is set to an
+kernel : :class:`~openturns.Distribution`, optional
+    A distribution over :math:`[0,1]^n`. If not mentioned, the kernel is set to an
     :class:`~openturns.IndependentCopula` with the same dimension as
     *distributions*.
 
 Notes
 -----
 A JointDistribution is a :math:`n`-dimensional distribution which can be
-written in terms of 1-d marginal distribution functions and a copula :math:`C`
-which describes the dependence structure between the variables.
-Its cumulative distribution function :math:`F` is defined by its marginal
-distributions :math:`F_i` and the copula :math:`C` through the relation:
+written in terms of 1-d distribution functions and a kernel :math:`K`
+which describes the interaction between the variables. If :math:`K` is a
+copula, then it represents exactly the dependence structure of the joint
```

Comment:
In mathematics, an object represents or does not represents something. It never _approximately_ represents something.
```suggestion
copula, then it represents the dependence structure of the joint
```